examples/00-fluent/exhaust_system_settings_api.py

```diff
@@ -52,6 +54,14 @@
     "exhaust_system.fmd", "pyfluent/exhaust_system"
 )
 
+###########################################################################
+# Specifying save path
+# ====================
+# * save_path can be specified as Path("E:/", "pyfluent-examples-tests") or
+# * Path("E:/pyfluent-examples-tests") in a Windows machine for example,  or
+# * Path("~/pyfluent-examples-tests") in Linux.
+save_path = Path(pyfluent.EXAMPLES_PATH)
```

Comment:
@seanpearsonuk 

We will use current working directory as save path in all examples. We have tried to use pyfluent.EXAMPLES_PATH as save path but GitHub runners shows it as invalid path due to permission issue.

https://github.com/ansys/pyfluent/actions/runs/13024727036/job/36331798117